### PR TITLE
Pixi v7 support and migrated to Extension Scripts

### PR DESCRIPTION
**Pull request by miltoncandelero**
Changed:
- Added V7 (`Assets`) support
- Removed support for pixi animate `v1` assets
- Updated build to use Extension Scripts
  - Updated Tests to run with Jest (Extension Scripts)
  - Updated docs to use webdoc (Extension Scripts)

Version number hasn't been bumped yet, However using `xs` to do so should be fairly trivial now

### CODE REVIEW

**Comment by andrewstart** [comment]
Does example.api.json and its MD5 hash need to be checked into the repo? Also, conflict in package-lock.json. Otherwise, this looks good.

**Comment by miltoncandelero** [comment]
Fixed both issues 👍

File: package.json [hunk 1]
```diff
@@ -2,99 +2,46 @@
...
-    "rollup-plugin-sourcemaps": "^0.4.2",
-    "rollup-plugin-ts": "^1.3.7",
-    "semver": "^5.3.0",
-    "terser": "^4.8.1",
-    "tslib": "^1.9.3",
-    "typedoc": "^0.23.17",
-    "typescript": "^3.9.0"
+    "@pixi/extension-scripts": "^1.7.0",
+    "pixi.js": "^7.1.1",
+    "typedoc": "^0.23.24",
```
Comment (bigtimebuddy) [inline package.json:28]: You don't need this if you're using webdoc in extension-scripts, right?

File: package.json [hunk 2]
```diff
@@ -2,99 +2,46 @@
...
+    "typescript": "^4.9.0"
```
Comment (bigtimebuddy) [inline package.json:29]: Not necessary to define since it's included in extension-scripts dependencies.

**Review by bigtimebuddy — APPROVED** [review]
Minor comments, but package.json looks good.

**Comment by miltoncandelero** [comment]
@bigtimebuddy Like this? 🤔

**Pull request merged**